apps/docs/docs/reference/ui/atoms/Price.mdx

```diff
@@ -1,15 +1,30 @@
 ---
 description: Prices are used to display the price of a given product, discount and total values.
-sidebar_custom_props: 
+sidebar_custom_props:
   image: https://vtexhelp.vtexassets.com/assets/docs/src/Price___9acce93fd9399d7dcea4a3bb8b7de0c6.png
 ---
 
-import PropsSection from "@site/src/components/PropsSection/PropsSection";
+import PropsSection from '@site/src/components/PropsSection/PropsSection'
+import { TokenTable, TokenRow, TokenDivider } from '@site/src/components/Tokens'
 
 # Price
 
 Prices are used to display the price of a given product, discount and total values.
 
+## Overview
+
+```tsx live
+<div className="overviewSection">
+  <Price value={62.5} variant="selling" />
+  <Price value={62.5} variant="listing" />
+  <Price value={62.5} variant="spot" />
+  <Price value={62.5} variant="savings" />
+  <Price value={62.5} variant="installment" />
+</div>
+```
+
+---
+
 ## Import
 
 ```tsx
```

Comment:
Can you also add styles import here pls.


```
```tsx
import '@faststore/ui/src/components/atoms/Price/styles.scss'
```
```